gradle.properties

```diff
@@ -1,27 +1,28 @@
 projectVersion=1.0.0.BUILD-SNAPSHOT
 micronautDocsVersion=1.0.24
-micronautVersion=2.0.0
+micronautVersion=2.0.1
```

Comment:
Can't build if I use snapshot. Need instructions on how to setup repositories since 2.0.11 is latest build tools and they are missing from the snapshots repository on https://oss.jfrog.org/oss-snapshot-local. That one contains 2.0.9.BUILD-SNAPSHOT as latest.

Reply:
Fine to leave it at 2.0.1 for now

Reply:
OK thanks I'll push changes I have soon for an easier review.